pkgs/applications/misc/flashcards/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, gtk4
+, pkg-config
+, cmake
+, libadwaita
+, blueprint-compiler
+, python3
+, desktop-file-utils
+, gobject-introspection
+, wrapGAppsHook4
+}:
+
+stdenv.mkDerivation rec {
+  pname = "flashcards";
+  version = "unstable-2023-06-11";
+
+  src = fetchFromGitHub {
+    owner = "fkinoshita";
+    repo = "FlashCards";
+    rev = "31d2be1cd5ea96be5266aedf24f73e8ac8a42c35";
+    hash = "sha256-aPnXazBWO+fjRbn1BzCO+kuYPT3y7SwPM97HK44Pm5Y=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    cmake
```

Comment:
Are you sure `cmake` is needed? Meson supports it for finding dependencies but typically, `pkg-config` will be sufficient. See <https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/meson-cmake.md> for more info.